aspnetcore/fundamentals/minimal-apis/7.0-samples/todo-group/Program.cs

```diff
@@ -0,0 +1,173 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.AspNetCore.OpenApi;
+using Repository;
+using Data;
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+
+// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+// Add TodoRepository to the container.
+builder.Services.AddScoped<TodosRepo>();
+// Add NoteRepository to the container.
+builder.Services.AddScoped<NotesRepo>();
+// Add InMemoryDatabase to the container.
+builder.Services.AddDbContext<ApplicationDbContext>(options =>
+{
+    options.UseInMemoryDatabase("InMemoryDb");
```

Comment:
Can we use [SQLite in-memory](https://docs.microsoft.com/en-us/ef/core/testing/testing-without-the-database#sqlite-in-memory) instead? [Our docs](https://docs.microsoft.com/en-us/ef/core/providers/in-memory/?tabs=dotnet-core-cli) "generally discourage" using this provider even for testing.